components/Projects.js

```diff
@@ -10,38 +11,10 @@ export default function Projects() {
         </h1>
       </div>
       <div className="bg-[#F1F1F1] dark:bg-gray-900 p-10 grid grid-cols-1 md:grid-cols-2">
-        {userData.projects.map((project, index) => (
-          <ProjectCard projectID={index} />
+        {userData.projects.map((project) => (
+          <ProjectCard key={project.title} project={project} />
```

Comment:
I suppose this is the preferred way to do this (passing an entire object in vs. passing in a way to reference it elsewhere). Why is this? Just to keep the code cleaner?